make/check_kqueue.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
```

Comment:
It was missing in the files in master too. Should I file PR to add it there too?